sdks/wasm/test.js

```diff
@@ -110,31 +114,20 @@ if (!send_message)
 
 print ("-----LOADED ----");
 
-var res = call_method (send_message, null, [mono_string ("run"), mono_string ("mini")])
-if (res)
-	print ("TEST RUN: " + conv_string (res))
-
-do {
-	res = conv_string (call_method (send_message, null, [mono_string ("run"), mono_string ("gc")]))
-	Module.pump_message ();
-} while (res == "IN PROGRESS");
-print ("DONE")
-
-
-var res = call_method (send_message, null, [mono_string ("say"), mono_string ("hello")])
-res = conv_string (res);
-if (res != "OK:3")
-	throw 4;
-
-var res = call_method (send_message, null, [mono_string ("say"), mono_string ("js-exception")])
-res = conv_string (res);
-if (res != "EH:1")
-	throw 5;
-
-try {
-	call_method (send_message, null, [mono_string ("say"), mono_string ("sharp-exception")])
-	print ("no exception??");
-	throw 6;
-} catch (e) {
+// var res = mono_send_msg ("start-test", "mini")
+// print ("-----STARTED ---- " + res);
+//
+// while (mono_send_msg ("pump-test", "") != "DONE") {
+// 	Module.pump_message ();
+// }
+//
+
+var res = mono_send_msg ("start-test", "corlib")
+print ("-----STARTED ---- " + res);
```

Comment:
Long term, how do we intend to report these errors? I worry that some browser terminals aren't prepared to handle the many megabytes of test log that we output during a normal BCL run. It might be more ergonomic, long-term, to have the browser download a file with the test output.

Reply:
Unfounded, this runs just fine on chrome. We might have to do other things to be able to run this on browsers and not on D8, as it's currently the case.